### PR TITLE
[SELinux lab2] Incorporate #155 feedback

### PR DESCRIPTION
**Pull request by wrabcak**
Added feedback from #155 . 

@dahaic please review.

### CODE REVIEW

File: 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc [hunk 1]
```diff
@@ -522,14 +496,12 @@ In this section, we are going to test our pre-configured setup steps from before
...
 
-	[root@selinux2 ~]# exit
-
 === Lab 2.3.3 SELinux Troubleshooting
 In the previous step, we turned SELinux on and got AVC denials. Denial messages are logged when SELinux denies access. Let's find out why we got these AVC denials.
 
 SELinux Troubleshooting can be performed on both the _selinux2_ and _selinux3_ hosts. We will use the _selinux2_ host for the following examples.
 
-. Log into the _selinux2_ host.
+. Log into the _selinux2_ host, if you're not already login.
```
Comment (dahaic) [inline 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc]: not already logged in.

File: 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc [hunk 2]
```diff
@@ -459,38 +451,21 @@ In this section, we are going to test our pre-configured setup steps from before
...
 
-    [root@selinux1 selinux_scripts]# curl selinux5
-    <h1>Default Document Root</h1>
-
-    [root@selinux1 selinux_scripts]# curl selinux5:7070
-    <h1>Custom Document Root</h1>
-
-	[root@selinux1 selinux_scripts]# ssh root@selinux2
-	[root@selinux2 ~]# getenforce
+	[root@selinux1 selinux_scripts]# ssh root@selinux{2,3,5} getenforce
```
Comment (dahaic) [inline 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc]: Unfortunately this won't work

File: 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc (hunk 1 above)
Comment (wrabcak) [reply 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc]: Fixed.

File: 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc (hunk 2 above)
Comment (wrabcak) [reply 2019Labs/RHELSecurityLab/documentation/lab2_SELinux.adoc]: lol, yes

**Comment by wrabcak** [comment]
@dahaic , updated based on your review.

**Comment by dahaic** [comment]
Super c00l!

**Pull request merged**